_posts/plotly_js/fundamentals/config-options/2020-01-24-display-modebar.html

```
@@ -0,0 +1,21 @@
+---
+name: Display Modebar
```

instead of "hide" and "display" the modebar can we find some way of saying "Disable modebar" and "Force modebar to be always visible" with an explanation of the default behaviour and what this override does?

I say this because the modebar already does "hide" and "display" by default, so just saying "hide" and "display" isn't quite clear enough.

---

We could also just restore the old title with the "always" for now and fix it later :)